apps/indexer/ponder.schema.ts

```diff
@@ -83,6 +83,8 @@ export const votingPowerHistory = onchainTable(
     votingPower: drizzle.bigint("voting_power").notNull(),
     delta: drizzle.bigint("delta").notNull(),
     timestamp: drizzle.bigint().notNull(),
+    // the voting power change non-negative to correlate with the transfer and delegation
+    deltaMod: drizzle.bigint("delta_mod").notNull(),
   }),
   (table) => ({
```

Comment:
The primary key definition is missing the transactionHash column. Since transactionHash is now nullable but still part of the primary key array, this will cause database constraint violations when transactionHash is null.